bootstrap

```diff
@@ -1,8 +1,8 @@
 #! /bin/sh
-set -ex
+# set -ex
```

Comment:
I there a reason for disabling this? `set -e` should be in every shell script IMO to prevent a continuation after error.